src/components/Page.tsx

```diff
@@ -5,22 +5,27 @@ import { pageContext } from '../utils/pageContext';
 import { ExternalPageProps } from '../utils/ssrProps';
 
 import { Header } from './Header';
+import { Footer } from './Footer';
+import styled from 'styled-components';
 
 interface PageProps {
     locale: ExternalPageProps['locale'];
     title: string;
 }
 
+const StyledContent = styled.div`
+    min-height: calc(100vh - 160px);
```

Comment:
Would be better to add comment about 160px. The next collaborator will not understand what is it.

Reply:
done